File: pinyougou_parent/pinyougou_manager_web/src/main/webapp/js/service/sellerService.js
app.service('sellerService', function ($http) {
    //旧的查询方法(弃置)								   
    this.findPage = function (pageNum, pageSize) {
        return $http.get('../seller/findPage.do?pageNum=' + pageNum + '&pageSize=' + pageSize)
    };

    //查询方法,读取列表数据绑定到表单中
    this.search = function (pageNum, pageSize, searchEntity) {
        return $http.post('../seller/findPage.do?pageNum=' + pageNum + "&pageSize=" + pageSize, searchEntity)
    };

    //删除方法
    this.removeSellerList = function (idList) {
        return $http.get('../seller/removeSeller.do?idList=' + idList)
    };

    //根据id查询
    this.findOne = function (id) {
        return $http.get('../seller/findOne.do?id=' + id)
    };

    //修改品牌
    this.updateSeller = function (seller) {
        return $http.post('../seller/updateSeller.do', seller)
    };

    //新增品牌
    this.saveSeller = function (seller) {
        return $http.post('../seller/saveSeller.do', seller)
    };

    //更改用户状态
    this.update = function (id, status) {
        return $http.get('../seller/update.do?sellerId='+id+'&status='+status);
    };
});

File: pinyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/SellerService.java
package com.pinyougou.sellergoods.service;
import java.util.List;
import com.pinyougou.pojo.TbSeller;							  
import entity.PageResult;
import entity.Result;
/**
 * seller服务层接口
 * @author jieweili
 *
 */
public interface SellerService {

    /**
     * 返回全部列表(已弃置)
     * @return
     */
    List<TbSeller> findAll();
 
    /**
     * 返回分页对象(过时)
     * @param pageNum
     * @param pageSize
     * @return
     */
	 
    PageResult findPage(Integer pageNum, Integer pageSize);

    /**
     * 添加数据
     * @param seller
     * @return
     */
    public Result saveSeller(TbSeller seller);

    /**
     * 根据id查询一个对象
     * @param id
     * @return
     */
    TbSeller getSellerById(String id);

    /**
     * 修改对象
     * @param seller
     * @return
     */
    Result updateSeller(TbSeller seller);

    /**
     * 批量删除
     * @param idList
     * @return
     */
    Result removeSeller(String[] idList);

    /**
     * 根据条件分页查询
     * @param seller
     * @param pageNum
     * @param pageSize
     * @return
     */
    PageResult listSellerByExample(TbSeller seller, Integer pageNum, Integer pageSize);


    /**
     * 修改状态吗
     * @param id
     * @param status
     * @return
     */
    Result update(String id,String status);
}
